src/redirect_traffic.js

```diff
@@ -0,0 +1,4 @@
+// redirect traffic to HTTPS from HTTP
+if (location.hostname != 'localhost' && location.hostname != 'localhost' && location.protocol != 'https:') {
```

Comment:
oops sorry, actually I was thinking of checking it for localhost as well as for 127.0.0.1 Should I edit it, and send a new PR?